lib/utils/HttpHelper.dart

```diff
@@ -29,10 +29,10 @@ class HttpHelper {
   static const String FROM = 'anonymous';
 
   /// Adds user agent data to parameters, for statistics purpose
-  static Map<String, String>? addUserAgentParameters(
-    Map<String, String>? map,
+  static Map<String, dynamic>? addUserAgentParameters(
+    Map<String, dynamic>? map,
   ) {
-    map ??= <String, String>{};
+    map ??= <String, dynamic>{};
```

Comment:
Why dynamic

Reply:
With the new method `doPatchRequest` the input can be `Map<String, dynamic>`, and not `Map<String, String>`.
More specifically in that case:
```dart
    parameterMap['product']['packagings'] = packagings;
```